src/pages/PageNotFound.jsx

```diff
@@ -1,9 +1,41 @@
 
+import Button from '../components/Button';
+
 const PageNotFound = () => {
     return (
-        <>
-            <h1>Page Not Found</h1>
-        </>
+        <div className="flex items-center justify-center h-5/6">
+            <div>
+                <h1 className="flex items-center justify-center h-5/6 text-6xl mb-2 font-bold" >Page Not Found</h1>
+           
+                <h2 className="flex items-center justify-center text-3xl mb-2 font-normal" style={{
+                    paddingTop: '20px'
+                }}> The page you're looking for doesn't exist.</h2>
+                <div className='flex items-center justify-center space-x-5 whitespace-nowrap font-bold pt-8'> 
+                    <button className="p-2 bg-neutral-800 px-5 text-white" style={{
+                        borderRadius: '10px'
+                    }} onClick={
+                        () => {
+                            window.location.href = "/"; // Redirect to home page
+                        } }>
+                        Return Home
+                    </button>
+                    <button className="p-2 px-5 " style={{
+                        borderRadius: '10px',
+                        borderWidth: '1px',
+                        borderColor: 'black'
+                    }} onClick={
+                        () => {
+                            window.location.href = "/"; // Redirect to home page
+                        } }>
+                        Get Help
+                    </button>
```

Comment:
nit: change to use the Button component. The styling won't look exactly correct right now, but if you notice in the Figma the buttons look pretty much the same across the site, and it's much easier to style one component than to style all the buttons on every page individually. For redirecting in the onClick using location.href works, but `useLocation()` is better supported so please switch to that